src/Lecture1.hs

```diff
@@ -1,126 +1,145 @@
-{- |
-Module                  : Lecture1
-Copyright               : (c) 2021-2022 Haskell Beginners 2022 Course
-SPDX-License-Identifier : MPL-2.0
-Maintainer              : Haskell Beginners 2022 Course <haskell.beginners2022@gmail.com>
-Stability               : Stable
-Portability             : Portable
-
-Exercises for the Lecture 1 of the Haskell Beginners course.
-
-To complete exercises, you need to complete implementation and add
-missing top-level type signatures. You can implement any additional
-helper functions. But you can't change the names of the given
-functions.
-
-Comments before each function contain explanations and example of
-arguments and expected returned values.
-
-It's absolutely okay if you feel that your implementations are not
-perfect. You can return to these exercises after future lectures and
-improve your solutions if you see any possible improvements.
--}
-
+-- |
+-- Module                  : Lecture1
+-- Copyright               : (c) 2021-2022 Haskell Beginners 2022 Course
+-- SPDX-License-Identifier : MPL-2.0
+-- Maintainer              : Haskell Beginners 2022 Course <haskell.beginners2022@gmail.com>
+-- Stability               : Stable
+-- Portability             : Portable
+--
+-- Exercises for the Lecture 1 of the Haskell Beginners course.
+--
+-- To complete exercises, you need to complete implementation and add
+-- missing top-level type signatures. You can implement any additional
+-- helper functions. But you can't change the names of the given
+-- functions.
+--
+-- Comments before each function contain explanations and example of
+-- arguments and expected returned values.
+--
+-- It's absolutely okay if you feel that your implementations are not
+-- perfect. You can return to these exercises after future lectures and
+-- improve your solutions if you see any possible improvements.
```

Comment:
It seems that you've used an autoformatter that changed the formatting of all comments in the file. With this, it's a bit difficult for me to review because it's not as easy to see the diff.

I can review this chapter for now but it would be easier for me to review solutions in the future without changing the format of comments 🙂

Reply:
Sorry. I've used VS Code Haskell extension and it has a default formatter. And I have format on save setting in VS Code. I'm used to using the autoformatter in other languages. I'll try to avoid reformatting all file for other lessons.